code/server/routes/api.js

```diff
@@ -127,8 +115,13 @@ function getProfileDate(req) {
 		dateOfBirth: req.body.dateOfBirth};
 }
 
+// Initialize as either 'db' or 'src'
+function initializeDAOs(mode) {
```

Comment:
What if we made the UserDAO and UserMatchDAO default to 'db' at the top of this file, and then kept this function so that it can be changed when the tests want it to be switched?

This would make it so we don't need main.js to know about the api at all, and it would prevent us from accidentally calling initializeDAOs